integration/src/main/resources/configs/integration/ListDatasets.json

```diff
@@ -13,5 +13,5 @@
       "expectedTimeForEachUnit": "SECONDS"
     }
   ],
-  "testUserFiles": [ "scarlett.json" ]
+  "testUserFiles": [ "user.json", "admin.json" ]
```

Comment:
I think doing this will cause the test to be run multiple times, once per test user. At this point I don't think there's a benefit to doing this, since the behavior will be the same for both users.

Reply:
I can remove the admin, np! I usually like to test everything cause the only sacrifice we make is time, and ... ya never know!